src/editor/draft.ts

```diff
@@ -1,17 +1,17 @@
 import { HoverNotification } from "../notification-system/notification";
-import { Expression } from "../syntax-tree/ast";
+import { Statement } from "../syntax-tree/ast";
 import { Module } from "../syntax-tree/module";
 
 export class DraftRecord {
-    code: Expression;
+    code: Statement;
     warning: HoverNotification;
 
     private module: Module; //no point in instantiating the editor itself because it will require an instance of Module anyway
 
-    constructor(code: Expression, module: Module) {
+    constructor(code: Statement, module: Module, txt?: string) {
```

Comment:
Necessary change since `FunctionCallStmt` objects can also be an `Importable`, which means they need to support DraftMode.